modules/gcloud/src/main/java/org/testcontainers/containers/SpannerEmulatorContainer.java

```diff
@@ -25,4 +25,20 @@ public SpannerEmulatorContainer(final DockerImageName dockerImageName) {
                 .withRegEx(".*Cloud Spanner emulator running\\..*"));
     }
 
+    /**
+     * @return a <code>host:port</code> pair corresponding to the address on which the emulator's
+     * gRPC endpoint is reachable from the test host machine. Directly usable as a parameter to the
+     * com.google.cloud.spanner.SpannerOptions.Builder#setEmulatorHost(java.lang.String) method.
+     */
+    public String getEmulatorGrpcEndpoint() {
+        return getContainerIpAddress() + ":" + getMappedPort(GRPC_PORT);
+    }
+
+    /**
+     * @return a <code>host:port</code> pair corresponding to the address on which the emulator's
+     * HTTP REST endpoint is reachable from the test host machine.
+     */
+    public String getEmulatorHttpEndpoint() {
+        return getContainerIpAddress() + ":" + getMappedPort(HTTP_PORT);
+    }
```

Comment:
This method has no test coverage, which I think is _OK_ but not ideal.

Given that AFAICT there's no practical reason to use the HTTP REST API from Java, I felt it was a bit 'meh' to have the method at all, but I included it for completeness.

Related to that, is the Javadoc comment fair and accurate? I avoided saying anything about how the resulting value could be used 😂